capstone/src/main/java/com/google/univiz/scorecard/CollegeData.java

```diff
@@ -1,5 +1,6 @@
-package com.google.univiz;
+package com.google.univiz.scorecard;
```

Comment:
According to PR description, this should be com.google.univiz.api.representation.